src/subapps/projects/containers/InputsContainer.tsx

```diff
@@ -1,14 +1,95 @@
 import * as React from 'react';
+import { Button, notification } from 'antd';
+import { useNexusContext } from '@bbp/react-nexus';
 
 import { useInputs } from '../hooks/useInputs';
 import InputsTable from '../components/InputsTable';
+import useLocalStorage from '../../../shared/hooks/useLocalStorage';
+import { DATASET_KEY } from '../../search/components/ResultGridActions';
+import { forceAsArray } from '../../../shared/utils';
 
 const InputsContainer: React.FC<{
   orgLabel: string;
   projectLabel: string;
   stepId: string;
 }> = ({ orgLabel, projectLabel, stepId }) => {
   const { inputs } = useInputs(orgLabel, projectLabel, stepId);
+  const [collection, setCollection] = useLocalStorage(DATASET_KEY);
+  const nexus = useNexusContext();
+
+  React.useEffect(() => {
+    if (!collection || !collection?.ids.length) {
+      return;
+    }
+
+    const key = `${orgLabel}-${projectLabel}-${stepId}`;
+
+    const createInputInWorkflowStep = async () => {
+      const workflowStep = await nexus.Resource.get<{
```

Comment:
Since we are using async/await, we may also want to put this inside a try catch block and handle the errors.